### PR TITLE
Jd/add pointers correctly

### PR DESCRIPTION
**Pull request by jd-lara**

### CODE REVIEW

File: forecasts/5bus_ts/7day/timeseries_pointers_agc_7day.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [
     {
         "simulation": "AGC",
+        "resolution": 4,
```
Comment (daniel-thom) [inline forecasts/5bus_ts/7day/timeseries_pointers_agc_7day.json:4]: Is this actually correct?

Comment (jd-lara) [reply forecasts/5bus_ts/7day/timeseries_pointers_agc_7day.json:4]: AGC is 4 seconds data

**Review by daniel-thom — COMMENTED** [review]
Looks good

**Comment by jd-lara** [comment]
> Looks good

Approval?

**Review by daniel-thom — APPROVED** [review]

**Pull request merged**